src/main/kotlin/com/vanniktech/android/apk/size/ApkSizePlugin.kt

```diff
@@ -0,0 +1,53 @@
+package com.vanniktech.android.apk.size
+
+import com.android.build.gradle.AppExtension
+import com.android.build.gradle.LibraryExtension
+import com.android.build.gradle.api.BaseVariant
+import org.gradle.api.DomainObjectCollection
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+open class ApkSizePlugin : Plugin<Project> {
+  override fun apply(project: Project) {
+    when {
+      project.plugins.hasPlugin("com.android.application") -> {
```

Comment:
We can drop the `{` here since it's a one liner

Reply:
Ok.

Reply:
Fixed.